iphone/Maps/Classes/ClipboardMonitor.mm

```diff
@@ -0,0 +1,107 @@
+        });
+
+        // Make an API request
+        NSURL *apiURL = [NSURL URLWithString:[NSString stringWithFormat:@"https://url-un.kartikay-2101ce32.workers.dev/search?url=%@", copiedText]];
+        NSURLSessionDataTask *task = [[NSURLSession sharedSession] dataTaskWithURL:apiURL completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [window.rootViewController dismissViewControllerAnimated:YES completion:nil];
+            });
+
+            if (error) {
+                NSLog(@"API Request Error: %@", error);
+                NSString *errorMessage = @"Kindly make sure that your internet connection in turned on.";
+                [self displayErrorMessage:errorMessage inWindow:window];
+            } else {
+                NSError *jsonError;
+                NSString *responseString = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+                NSLog(@"API Response: %@", responseString);
+                NSDictionary *responseDict = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:&jsonError];
+                if (jsonError) {
+                    NSLog(@"JSON Parsing Error: %@", jsonError);
+                    NSString *errorMessage = @"Kindly make sure that your URL points to an address";
+                  [self displayErrorMessage:errorMessage inWindow:window];
+                } else {
+                    NSDictionary *urlDict = responseDict[@"url"];
+                    NSString *geoURL = urlDict[@"geo"];
+                    if (geoURL) {
+                        NSLog(@"Geo URL: %@", geoURL);
+                        dispatch_async(dispatch_get_main_queue(), ^{
+                            GetFramework().ShowMapForURL(geoURL.UTF8String);
+                        });
+                    }
+                }
+            }
+        }];
+        [task resume];
+    }
+}
+
+- (BOOL)isURL:(NSString *)text {
+    NSDataDetector *detector = [NSDataDetector dataDetectorWithTypes:NSTextCheckingTypeLink error:nil];
+    NSArray *matches = [detector matchesInString:text options:0 range:NSMakeRange(0, text.length)];
+
+    for (NSTextCheckingResult *match in matches) {
+        if (match.resultType == NSTextCheckingTypeLink) {
+            return YES;
+        }
+    }
+
+    return NO;
+}
+
+- (void)displayErrorMessage:(NSString *)errorMessage inWindow:(UIWindow *)window {
+    if (!self.isErrorPromptShown) {
+        self.isErrorPromptShown = YES;
+
+        dispatch_async(dispatch_get_main_queue(), ^{
+            UIAlertController *alertController = [UIAlertController alertControllerWithTitle:@"Redirection Failed" message:errorMessage preferredStyle:UIAlertControllerStyleAlert];
+            UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
```

Comment:
There is already a translation for the OK button in strings.txt, check how to use it in other mm files.

Reply:
Implemented the translation for "ok" via localisation in the code, I have checked for different languages and it works. Reference :- 
<img width="340" alt="OK_localisedString" src="https://github.com/organicmaps/organicmaps/assets/120778728/c18582d9-abe3-4b3b-938e-6674ef3739ab">